ghost/core/core/server/data/migrations/versions/5.21/2022-10-26-04-50-member-subscription-created-request-id.js

```diff
@@ -0,0 +1,73 @@
+const logging = require('@tryghost/logging');
+const ObjectId = require('bson-objectid').default;
+const {createTransactionalMigration} = require('../../utils');
+const DatabaseInfo = require('@tryghost/database-info');
+
+// This migration links together members_created_events and members_subscription_created_events
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        if (DatabaseInfo.isSQLite(knex)) {
```

Comment:
Is there a specific reason we're skipping doing this on SQLite? Just curious

Reply:
`TIMESTAMPDIFF` isn't supported on SQLite (we could probably search for a workaround), and I wasn't sure about 'conflict on insert' working properly over there. For non-production databases I couldn't really see a value in merging the events